spec/spec_helper.rb

```diff
@@ -22,20 +23,28 @@ def initialize
     clear
   end
 
-  def debug(message, log_context = {})
-    logged << { message: message, context: log_context, severity: 'DEBUG' }
+  def debug(message, **log_context)
+    super.tap do
+      logged << { message: message, context: log_context, severity: 'DEBUG' }
+    end
```

Comment:
I wonder if we should do these with rspec mock `allow`, since it would check that the signature matches? The odds of changing the signature again aren't very high, though, so that might be overkill.

Reply:
Oh, I just noticed that since you call `super`, that actually will need to match the calling contract at that point. 💯 

But as a result, you'll wind up actually logging. Does that make the test output really noisy?